e2e/pages/collections/corrective-action/corrective-action.readonly.spec.js

```diff
@@ -1,31 +1,147 @@
 import CorrectiveActionPage from './corrective-action.po';
 import Hooks from '../../../utilities/hooks';
 
-let hooks, page;
+let hooks; let
+  page;
```

Comment:
```suggestion
let hooks;
let page;
```

I really don't like the way the auto-fix runs these into new lines like this, but don't think I have a way for the linting to enforce it